core/server/services/settings/index.js

```diff
@@ -4,16 +4,108 @@
  */
 
 var SettingsModel = require('../../models/settings').Settings,
-    SettingsCache = require('./cache');
+    SettingsCache = require('./cache'),
+    SettingsLoader = require('./loader'),
+    EnsureSettingsFiles = require('./ensure-settings'),
+    _ = require('lodash'),
+    common = require('../../lib/common'),
+    debug = require('ghost-ignition').debug('services:settings:index');
 
 module.exports = {
     init: function init() {
-        // Update the defaults
-        return SettingsModel.populateDefaults()
-            .then(function (settingsCollection) {
-                // Initialise the cache with the result
-                // This will bind to events for further updates
-                SettingsCache.init(settingsCollection);
+        const knownSettings = this.knownSettings();
+
+        debug('init settings service for:', knownSettings);
+
+        // Make sure that supported settings files are available
+        // inside of the `content/setting` directory
+        return EnsureSettingsFiles(knownSettings)
```

Comment:
@AileenCGN The CLI was not yet released. We can't release this without the CLI.
Furthermore, we can't release this **without requiring a minimum CLI version** (see package.json), because the settings folder must exist.

> without requiring a minimum CLI version

Can you please bump this in this PR and add a detailed explanation to the commit?
Please see also: https://github.com/TryGhost/Ghost-CLI/pull/681#issuecomment-378824798

Thanks!

Reply:
Yes, sorry I missed added this todo, but I had it in my head the whole time. I pushed the commit with the minimum CLI requirement to this PR.

> Please see also: TryGhost/Ghost-CLI#681 (comment)

What exactly from this comment are you referring to?

Reply:
@kirrg001 I submitted a PR to the CLI to make the error message a bit clearer: https://github.com/TryGhost/Ghost-CLI/pull/702